sermant-integration-tests/dubbo-test/dubbo-2-7-integration-provider/src/main/resources/rule.yaml

```diff
@@ -0,0 +1,215 @@
+servicecomb:
+  matchGroup:
+    demo-lb: |
+      matches:
+        - serviceName: dubbo-integration-provider
+    demo-fault-null: |
+      matches:
+        - apiPath:
+            exact: "io.sermant.integration.service.FlowControlService.faultNull"
```

Comment:
This file exists in consumer. What is the usage here?

Reply:
To test the limiting and circuit breaker capabilities of the server in the dubbo scenario